classes/class.ilObjOpenCastAccess.php

```diff
@@ -110,32 +110,28 @@ class ilObjOpenCastAccess extends ilObjectPluginAccess
     ];
 
 
-    /**
-     * @param string $a_cmd
-     * @param string $a_permission
-     * @param int    $a_ref_id
-     * @param int    $a_obj_id
-     * @param string $a_user_id
-     */
-    public function _checkAccess($a_cmd, $a_permission, $a_ref_id, $a_obj_id = null, $a_user_id = ''): bool
+
```

Comment:
The extra line should be removed or even better, replace it with inheritDoc:
```suggestion
    /**
     * @inheritDoc
     */
```